src/gc/impl/conservative/gc.d

```diff
@@ -1201,6 +1201,17 @@ class ConservativeGC : GC
     }
 
 
+    core.memory.GC.ProfileStats profileStats() nothrow
+    {
+        typeof(return) ret;
+
+        ret.numCollections = numCollections;
+        ret.totalCollectionTime = (prepTime + markTime + sweepTime + recoverTime).total!"hnsecs";
+        ret.maxCollectionTime = maxPauseTime.total!"hnsecs";
```

Comment:
collection time and pause time are not the same. The latter is the time that all other threads are suspended aka the world is stopped.